packages/ui/src/components/Label/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { AsteriskIcon } from '@ultraviolet/icons'
+import type { InputHTMLAttributes, ReactNode } from 'react'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+type LabelProps = {
+  children?: string
+  labelDescription?: ReactNode
+  required?: boolean
+  size?: 'small' | 'medium' | 'large'
+  htmlFor?: string
+  id?: string
+} & Pick<InputHTMLAttributes<HTMLInputElement>, 'aria-labelledby'>
+
+/**
+ * Label is used inside all of our input components, but it can be used outside for design purposes
+ */
+export const Label = ({
+  children,
+  labelDescription,
+  required,
+  size = 'medium',
+  htmlFor,
```

Comment:
```suggestion
  size = 'large',
  htmlFor,
```

Better follow the same default value of other inputs, which is `large`